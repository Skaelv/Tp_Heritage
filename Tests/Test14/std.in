C C1 1 1 1
R RECT 23 23 45 34
OA OA1 RECT C1
OA OA2 RECT OA1 OA1 RECT C1
EXIT
