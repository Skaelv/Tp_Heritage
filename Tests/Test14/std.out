 # Entrez une commande
C C1 1 1 1
R RECT 23 23 45 34
OA OA1 RECT C1
OA OA2 RECT OA1 OA1 RECT C1
EXITOK
# New Object : C1
 # Entrez une commande
OK
# New Object : RECT
 # Entrez une commande
OK
# New Object : OA1
 # Entrez une commande
OK
# New Object : OA2
 # Entrez une commande
